torch/onnx/symbolic_helper.py

```diff
@@ -867,6 +881,40 @@ def _set_training_mode(training_mode):
     global _training_mode
     _training_mode = training_mode
 
+# Symbolic function state.
+class SymbolicFunctionState(object):
+    # Skip check due to cannot import IValue from torch._C
+    params_dict = None  # type: ignore[var-annotated]
+    env = None  # type: Optional[Dict[torch._C.Value, torch._C.Value]]
```

Comment:
please use python 3 style type hints, not comments

Reply:
curious what's the rationale? I added python 3 style first but noticed that it was not supported for python < 3.6.

Reply:
It's a more modern, more compact way to write the same thing.
I think PyTorch requires Python >= 3.6 (or at least CI only tests back to 3.6), so there's no reason to use the old comment style.

Reply:
linters and static code analyzer do use type hints. Maybe they also do it for comment style, but I am not sure